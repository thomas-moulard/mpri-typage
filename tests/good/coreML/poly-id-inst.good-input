val id x = x
val z = id 0
val z' = id "foo"
