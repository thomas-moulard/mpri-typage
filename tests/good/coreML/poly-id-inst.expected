id = fun x.x

z = 0

z' = "foo"

z' : string
z : int
id : forall 'a.('a -> 'a)
